data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/AggregateGroup.java

```diff
@@ -62,6 +82,7 @@ boolean shouldConcludeGroup(final Duration groupDuration) {
 
     void resetGroup() {
         groupStart = Instant.now();
+        this.eventHandle = new AggregateEventHandle(groupStart);
```

Comment:
Should this do something with the old `eventHandle`? Release it? Something else?

Reply:
No. The previous event handle was s ent with the concluded event and will be released after the event is sent to Sink.

Reply:
Thanks for the explanation. This is clear. To be sure, are there any paths that could lead to resetting a group where we do need to handle the aggregate event?

Reply:
It doesn't look like it. It looks like resetGroup() is called only in closeGroup() which is called only when the group is concluded.